<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgImage</class>
 <widget class="QDialog" name="QG_DlgImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>331</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="lLayer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Layer:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QG_LayerBox" name="cbLayer" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QG_WidgetPen" name="wPen" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="buttonGroup8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Geometry</string>
       </property>
       <layout class="QGridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="lWidth">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lePath">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leInsertY">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lHeight">
          <property name="text">
           <string>Height:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lInsertX">
          <property name="text">
           <string>insert (x):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lScale">
          <property name="text">
           <string>Scale:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <spacer name="spacer58">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="leSize">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="lSize">
          <property name="text">
           <string>Size (px):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leInsertX">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lPath">
          <property name="text">
           <string>path:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <spacer name="spacer61">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lInsertY">
          <property name="text">
           <string>insert (y):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="leScale">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lAngle">
          <property name="text">
           <string>Angle:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leWidth">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leHeight">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="leAngle">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="lDPI">
          <property name="text">
           <string>DPI</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="leDPI"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QG_WidgetPen</class>
   <extends>QWidget</extends>
   <header>qg_widgetpen.h</header>
  </customwidget>
  <customwidget>
   <class>QG_LayerBox</class>
   <extends>QWidget</extends>
   <header>qg_layerbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leInsertX</tabstop>
  <tabstop>leInsertY</tabstop>
  <tabstop>leWidth</tabstop>
  <tabstop>leHeight</tabstop>
  <tabstop>leScale</tabstop>
  <tabstop>leAngle</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>leSize</tabstop>
  <tabstop>leDPI</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leWidth</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>changeWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leHeight</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>changeHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leScale</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>changeScale()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDPI</sender>
   <signal>editingFinished()</signal>
   <receiver>QG_DlgImage</receiver>
   <slot>changeDPI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeWidth()</slot>
  <slot>changeHeight()</slot>
  <slot>changeScale()</slot>
  <slot>changeDPI()</slot>
 </slots>
</ui>
